balena/models/config_variable.py

```diff
@@ -49,11 +49,9 @@ def get_all(self, uuid):
 
         """
 
-        device = self.device.get(uuid)
-
         params = {
-            'filter': 'device',
-            'eq': device['id']
+            'filter': 'device/uuid',
+            'eq': uuid
```

Comment:
Would it make a real difference if we changed it to use:
```
raw_query = '$filter=device/any(d:d/uuid%20eq%20{uuid})'.format(uuid=uuid)
```
?

Reply:
I think using raw query strings if not necessary makes it harder to maintain, no?
It's the same pattern of using pine client in node rather than writing the URLs by hand.

Reply:
@fisehara I don't think the python SDK's "lite pine client" implementation supports using a $any, and that's why I suggested to use raw_query. The existing code already does that in all cases that a $any was necessary, and a proper python pine client is a longstanding TODO.